server/src/lib/quotes.ts

```diff
@@ -41,6 +54,32 @@ class Quotes {
 
     return this.quotes[randomIndex];
   }
+
+  /**
+   * Returns groupings of quotes by author
+   *
+   * @returns {Array[Array<Quote>]}
+   */
+  public static getGroup(): Quote[][] {
```

Comment:
```suggestion
  public static getGroup(groups: number): Quote[][] {
```
^ What if we provided a parameter so when the user asks for a specific number of paragraphs we can control it.

Reply:
Nice